### PR TITLE
Refactor expression.go to improve readability

### PR DESCRIPTION
**Pull request by rogerlucena**
This PR comes to refactor `expression.go`, mainly renaming some very short variable names to make them more readable and clear for a new developer looking at the file.

### CODE REVIEW

File: bql/semantic/expression.go [hunk 1]
```diff
@@ -109,37 +109,39 @@ func formatCell(c *table.Cell) (string, error) {
 type evaluationNode struct {
 	op OP // operation.
 
-	lB string // left binding.
-	rB string // right binding.
+	leftBinding  string
+	rightBinding string
```
Comment (rbkloss) [inline bql/semantic/expression.go:113]: Thank you! =]

File: bql/semantic/expression.go [hunk 2]
```diff
@@ -81,6 +81,15 @@ func (o OP) String() string {
 	}
 }
 
+// cellFromRow retrives the value of a binding from a given row.
```
Comment (thiagovas) [inline bql/semantic/expression.go:84]: Little typo here:

s/retrives/retrieves

Comment (rogerlucena) [reply bql/semantic/expression.go]: Thank you Thiago, I have just corrected it.

File: bql/semantic/expression.go [hunk 3]
```diff
@@ -81,6 +81,15 @@ func (o OP) String() string {
 	}
 }
 
+// cellFromRow retrieves the value of a binding from a given row.
+func cellFromRow(binding string, r table.Row) (*table.Cell, error) {
+	val, ok := r[binding]
+	if !ok {
+		return nil, fmt.Errorf("comparison operation requires the binding value for %q for row %v to exist", binding, r)
+	}
```
Comment (rbkloss) [inline bql/semantic/expression.go:89]: This error seems to be out of scope for the function,
maybe return binding not found, and in the caller of this we reutn this message in case of error.

**Review by rbkloss — APPROVED** [review]

File: bql/semantic/expression.go (hunk 3 above)
Comment (rogerlucena) [reply bql/semantic/expression.go:89]: Done, thank you Kloss.

File: bql/semantic/expression_test.go [hunk 4]
```diff
@@ -96,10 +96,10 @@ func TestEvaluationNode(t *testing.T) {
 	for _, entry := range testTable {
 		got, err := entry.eval.Evaluate(entry.r)
 		if !entry.err && err != nil {
-			t.Errorf("failed to evaluate op %q for %v on row %v with error: %v", entry.eval.(*evaluationNode).op.String(), entry.eval, entry.r, err)
+			t.Errorf("failed to evaluate op %q for %v on row %v with error: %v", entry.eval.(*evaluationNode).operation.String(), entry.eval, entry.r, err)
```
Comment (jlsotomayorm) [inline bql/semantic/expression_test.go:99]: should be ok to remove ".String()"

When an operand implements the method String, the method will be invoked to convert the object to a string.

File: bql/semantic/expression_test.go [hunk 5]
```diff
@@ -96,10 +96,10 @@ func TestEvaluationNode(t *testing.T) {
...
 		}
 		if want := entry.want; got != want {
-			t.Errorf("failed to evaluate op %q for %v on row %v; got %v, want %v", entry.eval.(*evaluationNode).op.String(), entry.eval, entry.r, got, want)
+			t.Errorf("failed to evaluate op %q for %v on row %v; got %v, want %v", entry.eval.(*evaluationNode).operation.String(), entry.eval, entry.r, got, want)
```
Comment (jlsotomayorm) [inline bql/semantic/expression_test.go:102]: same as above

**Review by jlsotomayorm — APPROVED** [review]

File: bql/semantic/expression_test.go (hunk 4 above)
Comment (rogerlucena) [reply bql/semantic/expression_test.go]: Well pointed. Thank you Jose, I have just done so.

File: bql/semantic/expression_test.go (hunk 5 above)
Comment (rogerlucena) [reply bql/semantic/expression_test.go]: Done. =]
